pkg/cri/server/container_create_windows_test.go

```diff
@@ -245,3 +250,179 @@ func TestHostProcessRequirements(t *testing.T) {
 		})
 	}
 }
+
+func isEqualStringArrays(values, expected []string) bool {
+	if len(values) != len(expected) {
+		return false
+	}
+
+	for i, x := range expected {
+		if values[i] != x {
+			return false
+		}
+	}
+	return true
+}
+
+func assertEqualsStringArrays(values, expected []string) error {
+	if !isEqualStringArrays(values, expected) {
+		return fmt.Errorf("expected %s, but found %s", expected, values)
+	}
+	return nil
+}
+
+func getCreateContainerTestDataForArgsEscapedTests(imgEntrypoint []string, imgCmd []string, command []string, args []string) (*runtime.ContainerConfig, *runtime.PodSandboxConfig, *imagespec.ImageConfig) {
```

Comment:
If the combining all these tests into one suggestion is taken we should probably just use getCreateContainerTestData and edit the imageConfig in the one test